src/validator.js

```diff
@@ -368,6 +383,11 @@
     Extended.prototype = Object.create( Assert.prototype );
     Extended.prototype.constructor = Extended;
 
+    // Copy all the static methods.
+    Object.keys( Assert ).forEach( function( key ) {
```

Comment:
My bad, totally missed the above comments. No problem to merge like that, and _to be safe_ specify in the readme that it works on node and the browser with modern browsers. We'll see if issues occurs and if we'll need to point out some polyfills to make it work. Feel free to merge then.